src/common.rkt

```diff
@@ -101,7 +101,11 @@
 ;; Simple floating-point functions
 
 (define (ordinary-float? x)
```

Comment:
Since these aren't always floating point values now, it would be better to rename this function. `ordinary-value?` seems like a good more-general name. We could also add a contract to keep the types straight.